### PR TITLE
Update Quantity prefix to q/ to match name

### LINKED ISSUE #152: Update Quantity prefix to q/ to match name

### PR DESCRIPTION
**Pull request by kaixuan477**
Close #152

### CODE REVIEW

**Review by awdse22 — COMMENTED** [review]
Shouldn't the user guide (or any other relevant documentation) be updated as well?

File: docs/UserGuide.md.rej [hunk 1]
```diff
@@ -3,6 +3,6 @@ diff a/docs/UserGuide.md b/docs/UserGuide.md	(rejected hunks)
  **Edit a person**     | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
  **Find person(s)**    | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
  **List all persons**  | `list`
--**Add an order**      | `addOrder INDEX i/ITEM d/DATE [c/QUANTITY]` <br> e.g., `addOrder 2 i/Lamp d/20-11-2024 c/3`
+-**Add an order**      | `addOrder INDEX i/ITEM d/DATE [q/QUANTITY]` <br> e.g., `addOrder 2 i/Lamp d/20-11-2024 q/3`
```
Comment (FooChao) [inline docs/UserGuide.md.rej:6]: what is this file even lol?

**Review by FooChao — APPROVED** [review]
LGTM!

File: docs/UserGuide.md.rej (hunk 1 above)
Comment (kaixuan477) [reply docs/UserGuide.md.rej:6]: Honestly no clue, but updated for standardization.

**Pull request merged**